src/driver.cpp

```diff
@@ -89,7 +89,10 @@ std::set<std::string> Driver::list_modules() const
     for (auto &ap : *probe->attach_points)
     {
       auto probe_type = probetype(ap->provider);
-      if (probe_type == ProbeType::kfunc || probe_type == ProbeType::kretfunc)
+      if (probe_type == ProbeType::kfunc || probe_type == ProbeType::kretfunc ||
+          ((probe_type == ProbeType::kprobe ||
+            probe_type == ProbeType::kretprobe) &&
+           !ap->target.empty()))
```

Comment:
Future work idea: we could eventually drop the last part of the condition and parse BTF also for module functions specified in the form `kprobe:function` (as we do for kfuncs).

Reply:
Yeah, that's a good idea too. Probably wouldn't be too tough, either.